.vscode/settings.json

```diff
@@ -6,4 +6,5 @@
 
     // Use 'prettier-eslint' instead of 'prettier'. Other settings will only be fallbacks in case they could not be inferred from eslint rules.
     "prettier.eslintIntegration": true,
+    "editor.insertSpaces": false,
```

Comment:
I was wondering if this is intentionally pushed. what is it doing? and is that sth other people would like to have too?

Reply:
I didn't want to push it, but I've been fighting with that setting for some time, since on other projects I use spaces.

I thought that since in this project we use tabs, it's safe to let this setting as shared.

No problem removing it if we better don't have it there.

Reply:
Sounds good but I think it'd be better if we could open a separate PR and make it more visible to everyone, wdyt?

Reply:
Sure!